### DISCUSSION TITLE
Is it possible to automatically move a note based on a change/addition in its fileClass?

### DISCUSSION THREAD

**Discussion by cameronj86** [mdelobelle/metadatamenu#582] [Q&A]
For example, let's say I have a folder for open tickets. I have three files classes, `backlog`, `in progress` and `complete`. I'd like to have it set up so that when I change the fileClass from `backlog` -> `in progress`/`complete`, the file is moved to a different folder. 

It's basically the reverse functionality of the `file path` option: instead of "all files in a folder inheriting a fileClass", it is "all file in a fileClass inheriting a folder".

Curious if this functionality exists or anyone knows a supplemental plugin that could help? My research so far shows that the 'auto move' functionality is limited mostly to file name. I use Linux so I could potentially scrape together a bash script on cron with a little hand holding.

**Comment by mdelobelle** [comment]
Hi this is not in the scope of the plugin, yet you could totally write a plugin for it.
Since the 0.8.0-beta the plugin triggers an event for each field change `metadata-menu:fields-changed` ( https://github.com/mdelobelle/metadatamenu/blob/refacto-to-bulk/src/typings/types.d.ts#L186 ) that you can register to to make your own special actions:


app.metadataCache.on("metadata-menu:fields-changed", (data)=>{
// move your file if data matches specific conditions
})

**Reply by cameronj86** [reply]
Thanks for the suggestion! I'll give it a shot and see what I can put together
